lms/resources/_js_config/__init__.py

```diff
@@ -41,7 +41,7 @@ def _h_user(self):
     def _application_instance(self):
         return self._lti_user.application_instance
 
-    def add_document_url(self, document_url):
+    def add_document_url(self, document_url) -> None:
```

Comment:
Methods that have type annotations in the body but not type signatures are not checked by mypy and raised as an error. 

We have this one and a few constructors.